tests/gitgitgadget.test.ts

```diff
@@ -215,8 +216,8 @@ Cc: Some Body <somebody@example.com>
 
     expect(patches.coverLetter).toEqual(`My first Pull Request!
 
-This Pull Request contains some really important changes that I would love to
-have included in git.git.`);
+This Pull Request contains some really important changes that I would love
+to have included in git.git [https://github.com/git/git].`);
```

Comment:
Why sometimes `git.git [https://github.com/git/git]` and others `[git.git](https://github.com,/git/git)` ?

Trying to understand the intention.

Reply:
These tests verify the output to be the plain text (with link targets in square brackets), and it needs to provide the input (the "Pull Request description") as Markdown.

So you see the link provided in Markdown *once*, and all the other times it is the text that tries to assure the output format is correct.